src/common/verify/isValidAccessGrant.ts

```diff
@@ -48,7 +51,20 @@ async function isValidAccessGrant(
   } = {},
 ): Promise<{ checks: string[]; warnings: string[]; errors: string[] }> {
   const fetcher = await getSessionFetch(options);
-  const vcObject = await getBaseAccess(vc, options);
+  let validVC = null;
```

Comment:
```suggestion
  let validVc = null;
```
That's minor, but introduces a naming inconsistency otherwise.